### DISCUSSION TITLE
Bug Report Batch Naming

### DISCUSSION THREAD

**Discussion by Ericmeister** [upscayl/upscayl#391] [Releases]
I had it do a batch of 8 frames and swaped the names of the last two. Original picture 7 was named 8 and vice versa. All the other pictures were named accordingly to the original file. 
Im on a MacBook M1 Max and used all the standard settings which it has after the installation in version 2.5.5

**Comment by aaronliu0130** [comment]
Please file an issue instead of a discussion for bug reports. While doing that, please send logs, the instructions are detailed [here](https://github.com/upscayl/upscayl/wiki/Guide#logs).
